aperturedb/EntityDataCSV.py

```diff
@@ -70,3 +72,43 @@ def getitem(self, idx):
     def validate(self):
         if self.header[0] != ENTITY_CLASS:
             raise Exception("Error with CSV file field: " + ENTITY_CLASS)
+
+# Used when a csv has a single entity type that needs to be deleted
+
+
+class SingleEntityDeleteCSV(CSVParser.CSVParser):
```

Comment:
Can we conform to *DataCSV for all specific CSV parsers. 
Also, The Data CSV must come with a docstring which outlines a sample CSV template.